examples/wms-custom-tilegrid-512x256.js

```diff
@@ -25,7 +25,6 @@ var layers = [
     source: new ol.source.MapQuest({layer: 'sat'})
   }),
   new ol.layer.Tile({
-    extent: [-13884991, 2870341, -7455066, 6338219],
```

Comment:
the layer extent is surely for a different purpose: client-side limitation rather than server-side; an app might allow a user to define a smaller extent for a layer than that provided by the server